core/src/lib.rs

```diff
@@ -7,6 +7,8 @@ use crate::meta::types::CustomType;
 
 pub mod meta;
 
```

Comment:
Issue: Please add module-level docs explaining what this code is for, including README and a description in Cargo.toml